webapp/src/components/team_sidebar/index.js

```diff
@@ -1,6 +1,6 @@
 import {connect} from 'react-redux';
 
-import TeamSidebar from './team_sidebar.jsx';
+import TeamSidebar from './team_sidebar.tsx';
```

Comment:
We can use this without the extension

```suggestion
import TeamSidebar from './team_sidebar';
```